ClassRegistration/ClassRegistration.Domain/Repositories/IGenericRepository.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using ClassRegistration.Domain.Model;
+using ClassRegistration.DataAccess.Entity;
+
+namespace ClassRegistration.Domain.Repositories
+{
+    interface IGenericRepository<TDAL, TBLL>
+        where TDAL : DataModel, new()
+        where TBLL : BaseBusinessModel, new()
+    {
+    }
+}
+
+
+    
+
```

Comment:
Not really needed, since GenericRepository will never have any explicitly created instances